src/Symfony/Component/Routing/RouteCollection.php

```diff
@@ -303,4 +338,55 @@ public function addResource(ResourceInterface $resource)
             $this->resources[$key] = $resource;
         }
     }
+
+    /**
+     * Sets an alias for an existing route.
+     *
+     * @param string       $alias The alias to create
+     * @param string|Alias $route The route to alias
```

Comment:
`$alias` is a string and `$route` is an instance of `Alias`. This is really confusing when reading the code of this method.

That being said: Do we actually need the possibility to pass an instance of `Alias` here?

Reply:
I don't think we need it, I just wanted to be consistent with DI's alias system.

Reply:
Done.